### PR TITLE
Wf algorithm

### PR DESCRIPTION
**Pull request by jeromekelleher**
Depends on #9.

Changes the WF algorithm to use TAOCP notation and adds a discussion.

I'm going to stop replacing the things that you've done now @petrelharp!

### CODE REVIEW

**Comment by petrelharp** [comment]
No worries!  As long as you keep replacing them with better versions!

How do I tell when this is ready to merge? When you submit the pull request?

**Comment by jeromekelleher** [comment]
> How do I tell when this is ready to merge? When you submit the pull request?

I don't want to just push into master without it going through you first, as PRs are a good forum to discuss things. If there is a PR and you like what you see, then feel free to merge it. If there is some specific changes you think would improve it then it's a good way to talk about these.

File: forwards_paper.tex [hunk 1]
```diff
@@ -260,79 +260,93 @@ \subsection*{Tree sequences}
...
+from $x$ to $L$ is $P_b$. We then complete the replacement event by incrementing
+$n$, ready to represent the next new node.
 
 
+\begin{enumerate}
+\item Explain how to add mutational information. We can also do this afterwards on the
+output trees if they are nuetral.
+\item Explain that we can input these trees into an mpsrime tree sequence and they will
+work fine. Perhaps a diagram to show the kind of trees we get? (Multiroots, lots of unary nodes.
+Can use the msprime SVG API to draw these?) We could refer ahead to figure 2
```
Comment (petrelharp) [inline forwards_paper.tex:344]: We could just refer ahead.  But it might be *rad* to have a side-by-side of a pair of pre- and post-simplify trees.

**Comment by petrelharp** [comment]
like

**Pull request merged**